eaf.el

```diff
@@ -1697,7 +1697,8 @@ For a full `install-eaf.py' experience, refer to `--help' and run in a terminal.
          (proc
           (progn
             (async-shell-command (concat
-                                  eaf-python-command " install-eaf.py" " --install "
+                                  eaf-python-command " install-eaf.py"
+                                  (when (length> apps 0) " --install ")
```

Comment:
There is a typo here at `length>`

Reply:
Oh, this is a new function in version 28, it should be compatible with the old version, my fault